CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.7.7
+- Added support for retrieving the Pubnub subscription details
```

Comment:
Turns out that pubnub has a bug that doesn't close previous connections once a new subscriptions is called. This isn't really a problem though because the last connection made to pubnub handles all of the prior subscriptions. So I don't think a change needs made to the current way I am implementing this in Home-Assistant.